src/components/AddPaymentMethodMenu.js

```diff
@@ -61,12 +78,21 @@ function AddPaymentMethodMenu(props) {
             onItemSelected={props.onClose}
             menuItems={[
                 {
-                    text: props.translate('common.bankAccount'),
+                    text: props.translate('common.personalBankAccount'),
                     icon: Expensicons.Bank,
                     onSelected: () => {
                         props.onItemSelected(CONST.PAYMENT_METHODS.BANK_ACCOUNT);
                     },
                 },
+                ...(ReportUtils.isIOUReport(this.props.iouReport) && ReportActionsUtils.hasRequestFromPayer(lodashGet(this.props.iouReport, 'reportID', 0), this.props.session.accountID)
```

Comment:
I'm a bit confused by `hasRequestFromPayer`. Wouldn't this be checking if the current user that is clicking the pay button as made the request?

Reply:
should be negated yes